cloudinit/config/cc_ubuntu_advantage.py

```diff
@@ -88,6 +95,32 @@
           - fips
         """
         ),
+        dedent(
+            """\
+        # Do not enable any service in Ubuntu Pro instances.
+        ubuntu_advantage:
+          enable: []
+          enable_beta: []
+        """
+        ),
+        dedent(
+            """\
+        # Enable esm and beta realtime-kernel services in Ubuntu Pro instances.
```

Comment:
Is it possible to inform the users, that including beta services in the `enable` list will cause an error?

Reply:
Done.